modules/n1ql/pages/n1ql-language-reference/join.adoc

```diff
@@ -170,6 +170,8 @@ This expression may contain fields, constant expressions, or any complex {sqlpp}
 
 * You may chain ANSI joins with comma-separated joins; however, the comma-separated joins must come after any JOIN, NEST, or UNNEST clauses.
 
+* INNER joins and LEFT OUTER joins support the LATERAL keyword in front of the right-hand side keyspace. RIGHT OUTER joins do not support the LATERAL keyword.
```

Comment:
Good start on the description, but this is a new keyword, not a limitation, so I don't think it belongs in this section. We aim to describe the keywords (as far as possible) in the order they appear in the syntax summary. So there should be a new heading 4 section called something like `==== LATERAL Joins` between the sections on Join Type and Join Predicate &mdash; i.e. starting at line 148 above.